File: C/circumference_of_circle.c
// C program to calculate and print the circumference of a circle with the given radius.

#include<stdio.h>
#define pi 3.14159265359

int main(){
  float radius, circumference;

  printf("Enter the radius of circle : ");
  scanf("%f", &radius);

  circumference = 2 * pi * radius;

  printf("The circumference of circle with radius %f is = %.2f", radius, circumference);

  return 0;
}

File: Python/circumference_of_circle.py
# Python program to calculate the circumference of a circle with the given radius.

radius = float(input("Enter the radius of circle : "))
PI = 3.14159265359
circumference = 2 * PI * radius

print("The circumference of circle with radius "+str(radius)+" is = "+str(circumference))

# We can also print the result using the f string or string literals as given below.
# print(f"The circumference of circle with radius {radius} is {circumference}")
